tests/test_calc_impls.py

```diff
@@ -37,11 +37,13 @@
                           antiferro_ef,
                           kagome_ferromagnet,
                           kagome_antiferromagnet,
-                          kagome_supercell,])
+#                          kagome_supercell,
+                          ])
 def test_calc_impls(example):
     """Compare Rust and Python spinwave calculation implementations."""
-    rs_results = rs_spinwave(*example(classes=rs_classes))
-    py_results = py_spinwave(*example(classes=py_classes))
+    rs_energies, rs_sqw = rs_spinwave(*example(classes=rs_classes))
+    py_energies, py_sqw = py_spinwave(*example(classes=py_classes))
 
     # we test to an absolute tolerance of 1e-6 in line with the MATLAB
-    np.testing.assert_allclose(np.sort(rs_results), np.sort(py_results), atol=1e-6, rtol=0)
+    np.testing.assert_allclose(np.sort(rs_energies), np.sort(py_energies), atol=1e-6, rtol=0)
+    np.testing.assert_allclose(np.sort(rs_sqw), np.sort(py_sqw), atol=1e-6, rtol=0)
```

Comment:
Finally I found that I needed to change `rtol=1e-3` here to get the `antiferro_chain` test to pass, even with all the above changes...